TYPO3.Neos/Classes/TYPO3/Neos/Controller/Backend/ContentController.php

```diff
@@ -258,7 +259,7 @@ protected function getImagePreviewData(Image $image)
                 'height' => $thumbnail->getHeight()
             ];
         }
-        $thumbnail = $this->thumbnailService->getThumbnail($image, $this->thumbnailService->getThumbnailConfigurationForPreset('TYPO3.Neos:InspectorImage'));
+        $thumbnail = $this->thumbnailService->getThumbnail($image, new ThumbnailConfiguration(null, 300, null, 300));
```

Comment:
still need to use a preset, otherwise it basically has the same effect of rendering a ton of thumbnails just for this purpose.. either use the existing 250 or 1000 preview